yarn.lock

```diff
@@ -4017,11 +4017,6 @@ acorn@^6.0.1, acorn@^6.4.1:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-6.4.1.tgz#531e58ba3f51b9dacb9a6646ca4debf5b14ca474"
```

Comment:
Strange the lockfile has changed, since no deps have changed.

Reply:
I do not understand why

Reply:
Off topic: Any particular reason not to drop `yarn` and choose `npm`?

Reply:
@robinpiets yarn supports workspaces. https://doppelmutzi.github.io/monorepo-lerna-yarn-workspaces/

Reply:
@jasperswart Perhaps it's good thing to discard these changes

Reply:
or we can merge https://github.com/Amsterdam/amsterdam-styled-components/pull/505 and rebase these changes onto master, so netlify can build :)